src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -80,15 +88,15 @@
         WHERE
           ((strategy_mutator_plugin = 0) OR (strategy_mutator_plugin IS NULL)) AND
           /* Query results from the past 30 days. Change as needed. */
```

Comment:
Nit: fix comment. A few others might need to be updated as well, such as "temperature parameter" (singular) above.